simularium/binding-sim-edu
# Feature/init image

Problem
=======
Estimated review size: small

closes #90 

Solution
========
Added the new image. I used z stacking to hide the sides to avoid resizing issues. 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)


Steps to Verify:
----------------
1. the initial view should have a full width image. 
2. when you click back to the lab view, it should be behind the sidebars 

Screenshots (optional):
-----------------------
<img width="842" alt="Screenshot 2024-06-05 at 9 25 39 AM" src="https://github.com/simularium/binding-sim-edu/assets/5170636/315b04a0-cf94-4987-98aa-a2a63ab31c83">
<img width="555" alt="Screenshot 2024-06-05 at 9 25 30 AM" src="https://github.com/simularium/binding-sim-edu/assets/5170636/ccfef860-3a48-4d3b-b49e-2f15afa90fa5">
